src/main/java/MyCallable.java

```diff
@@ -0,0 +1,15 @@
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class MyCallable implements Callable<Integer> {
```

Comment:
```suggestion
public class ListSumCalculator implements Callable<Integer> {
```
name your classes according to their purpose